test/gpu/native/studies/chai/SKIPIF

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+CWD=$(cd $(dirname ${BASH_SOURCE[0]}) ; pwd)
+CHAI_BRANCH=main
+CHAI_URL=${CHAI_URL:-https://github.com/chapel-lang/ChAI.git}
```

Comment:
Given that we can use our own fork, do we really need the patch infrastructure? We should be able to adjust our fork as we need.

Reply:
Good point, I can make some updates to chapel-lang's ChAI and remove that.